packages/salesforcedx-vscode-core/src/settings/sfdxCoreSettings.ts

```diff
@@ -94,6 +98,10 @@ export class SfdxCoreSettings {
     return this.getConfigValue(SETTING_CLEAR_OUTPUT_TAB, false);
   }
 
+  public getNodeExtraCaCerts(): string {
+    return this.getConfigValue(ENV_NODE_EXTRA_CA_CERTS, '');
```

Comment:
@CristiCanizales I was thinking that we could also consider env var for NODE_EXTRA_CA_CERTS as a default if the customer does use the setting.
```suggestion
    return this.getConfigValue(ENV_NODE_EXTRA_CA_CERTS, process.env.NODE_EXTRA_CA_CERTS ?? '');
```